microsoft/testsuites/network/common.py

```diff
@@ -128,7 +128,7 @@ def sriov_vf_connection_test(
 
     dest_ssh.enable_public_key(source_ssh.generate_key_pairs())
     # generate 200Mb file
-    source_node.execute("dd if=/dev/random of=large_file bs=1M count=200")
+    source_node.execute("dd if=/dev/zero of=large_file bs=1M count=200")
```

Comment:
I remembered it's changed to random last time, because of zeros are optimized by some layers, so it's hard to test right performance. What's the reason to change it back?

Reply:
in the ubuntu FIPS kernel, it hung on this command. 
I asked Wei for this, he said the random generated based on Entropy, and Entropy may depend on the interrupt, if the interrupts are not enough, then it is slow to generate the random.

Reply:
Is there a way to make it's not random, but not zero?

Reply:
I'll figure it out

Reply:
Can you try /dev/urandom?

Reply:
based on my search, both random and urandom rely on Entropy, but I will try

Reply:
It doesn't matter to rely on entropy or not. The problem is, if the entropy is not enough, random will block to wait, and urandom never blocks. urandom reuse the entropy pool.